README.md

```diff
@@ -1,34 +1,30 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
+**Before you initiate a pull request**, please read the process document. Ideas should be thoroughly discussed prior to opening a pull request, such as on the [Ethereum Magicians forums](https://ethereum-magicians.org) or in a GitHub issue in this repository.
 
-# Contributing
+This repository tracks the ongoing status of EIPs. It contains:
 
- 1. Review [EIP-1](EIPS/eip-1.md).
```

Comment:
Why have you deleted all this explanatory text?

Reply:
Currently the README.md is focused on telling people how to dump whatever questions or ideas are in their head as quickly as possible into an EIP, which is a proposal to tell everybody else about a change they should make.

The proposed README.md focuses instead on explaining what an EIP is and how people should be more disciplined before they think about starting one.

// Ideas stolen from https://github.com/apple/swift-evolution which has a very good change management process.